examples/v1beta1/file-metricscollector-example.yaml

```diff
@@ -58,7 +58,7 @@ spec:
                 imagePullPolicy: Always
                 command:
                   - "python"
-                  - "/var/mnist.py"
+                  - "/opt/mnist/src/mnist.py"
```

Comment:
I think we must also fixes examples for v1alpha3, because we are using the same image names for v1alpha3 and v1beta1 version. For this example it is: `docker.io/kubeflowkatib/pytorch-mnist:1.0`.

What do you think @gaocegege @johnugeorge ?

Reply:
SGTM